src/figuresClasses.ts

```diff
@@ -1,19 +1,111 @@
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+enum Color{
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
+}
+
+const INVALID_SIZE_ERROR = 'Invalid sides for this figure';
```

Comment:
This error does not fit circle - as circle has no sides

Reply:
Overal a better practice is being more friendly to your users and provide more descriptive error messages